include/itkObjectnessMeasureImageFilter.h

```diff
@@ -23,9 +23,30 @@
 namespace itk
 {
 /** \class ObjectnessMeasureImageFilter
+ * \brief Enhance M-dimensional objects in N-dimensional images.
  *
- * This is a composite filter which combines a computation of the
- * hessian with computation of the objectness.
+ * This filter is a generalization of Frangi's vesselness measurement
+ * for detecting M-dimensional object in N-dimensional space. For
+ * example a vessel is a 1-D object in 3-D space. The filter
+ * can enhance blob-like structures (M=0), vessel-like structures
+ * (M=1), 2D plate-like structures (M=2), hyper-plate-like structures
+ * (M=3) in N-dimensional images, with M<N.
+ *
+ * This filter takes a scalar image as input and produces an real
+ * valued image as output which contains the objectness measure at
+ * each pixel. Internally, it computes a Hessian via discrete central
+ * differences. It is recommended to apply a Gaussian smoothing filter
```

Comment:
I would suggest changing the wording to:
"Before applying this filter it is expected that a Gaussian smoothing filter at an appropriate scale (sigma) was applied to the input image." 

I don't think this is an optional step (unless you are lucky).